started/packaging.md

```diff
@@ -17,6 +17,12 @@ with Linux there are various metadata files that should get installed. What a ha
 
 Thankfully the "fyne" app has a "package" command that can handle this automatically. Just specifying the target OS and any required metadata (such as icon) will generate the appropriate package. The icon conversion will be done automatically for .icns or .ico so just provide a .png file :). All you need is to have the application already built for the target platform...
 
+```
+go install fyne.io/fyne/v2/cmd/fyne@latest
+fyne package -os darwin -icon myapp.png
+```
+If you're using an older version of Go (<1.14), you should install fyne using `go get`
```

Comment:
Hmm, wasn't the new syntax added in Go 1.16?

Reply:
Fixed with d0d1045c7dda6f4ffe3c8f2ff2cd4c645e25530c